Remotion/Web/Development.WebTesting.IntegrationTests/TestExecutionScreenshotRecorderTest.cs

```diff
@@ -94,40 +92,42 @@ public void TestExecutionScreenshotRecorderTest_TakeDesktopScreenshot_ReplacesIn
     }
 
     [Test]
```

Comment:
it looks like the method order got changed. could you please sort it in the original order, thanks!